src/pages/Categories.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+function Categories() {
+  return (
+    <div>Categories</div>
+  );
```

Comment:
### 🛑 Required Change
- [ ] The Categories component is meant to display a button with text "Check status" 🤷🏻

Reply:
Added